pkgs/applications/misc/obinskit/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchurl 
+, xlibs
+, libxkbcommon
+, libudev
+, gcc-unwrapped
+, electron_3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obinskit";
+  version = "1.1.1";
+
+  src = fetchurl {
+    url = "http://releases.obins.net/occ/linux/tar/ObinsKit_${version}_x64.tar.gz";
+    sha256 = "0052m4msslc4k9g3i5vl933cz5q2n1affxhnm433w4apajr8h28h";
+  };
+
+  unpackPhase = ''
+    tar -xzf $src
+    export src='ObinsKit_1.1.1_x64'
+  '';
+
+  installPhase = ''
+    mkdir -p $out/opt/obinskit
+    install $src/icudtl.dat $out/opt/obinskit/
+    install $src/natives_blob.bin $out/opt/obinskit/
+    install $src/v8_context_snapshot.bin $out/opt/obinskit/
+    install $src/blink_image_resources_200_percent.pak $out/opt/obinskit/
+    install $src/content_resources_200_percent.pak $out/opt/obinskit/
+    install $src/content_shell.pak $out/opt/obinskit/
+    install $src/ui_resources_200_percent.pak $out/opt/obinskit/
+    install $src/views_resources_200_percent.pak $out/opt/obinskit/
+    cp -r $src/resources $out/opt/obinskit/
+    cp -r $src/locales $out/opt/obinskit/
```

Comment:
By setting the sourceRoot all these files will be in $CWD so you can drop `$src/` everywhere.